<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgFingerprint</class>
 <widget class="QDialog" name="CDlgFingerprint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>330</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Register Fingerprint</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>314</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-3</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>9</y>
     <width>20</width>
     <height>311</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>8</y>
     <width>20</width>
     <height>311</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>601</width>
     <height>400</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
   <widget class="QPushButton" name="buttonBoxOk">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>250</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBoxCancel">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>250</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblFPrintMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Please press any finger firmly against the scanner's</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFPrintMessage1">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>65</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>glass and then remove it</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStageText">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>130</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFPrintStageNumber">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>135</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFPrintOf">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>135</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>of</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFPrintStageTotal">
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>135</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFPrintMessage2">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>200</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
